### PR TITLE
only check if keys are empty, not all fields in DS settings

### PR DESCRIPTION
**Pull request by birarda**
- fixed a bug adding rows in domain-server settings with default values

### CODE REVIEW

**Comment by birarda** [comment]
#### Test Plan

- On master (latest dev-download), add a try to add a persistent script to your domain-server settings with a blank pool (the default). It should fail and tell you there are empty fields.
- On this PR, do the same. It should succeed and allow you to add the script.

**Comment by OmegaHeron** [comment]
Tested Ubuntu 16.04.1

QA Pass
Verified bad behavior in master
Verified PR rectifies

Will check on Windows 10 once it builds here as well.

File: domain-server/resources/web/settings/js/settings.js [hunk 1]
```diff
@@ -908,31 +908,11 @@ function validateInputs() {
...
-    });
-
-    if (empty) {
-      showErrorMessage("Error", "Empty field(s)");
-      inputsValid = false;
-      return
-    }
-
     // validate keys specificially for spaces and equality to an existing key
     var newKeys = $(table).find('tr.' + Settings.NEW_ROW_CLASS + ' td.key');
```
Comment (huffman) [inline domain-server/resources/web/settings/js/settings.js]: It seems like most of the tables don't add the `key` class to the first cell - is this intentional?

Comment (birarda) [inline domain-server/resources/web/settings/js/settings.js]: Right - only tables that are not number indexed arrays use a key.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5bd8e465108f3ee386b4716ecc9b85718366e783 available at: https://gist.github.com/d66febb0ab3fbafa4fb2c3d49170eece<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9577-5bd8e46.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9577-5bd8e46.dmg

**Comment by OmegaHeron** [comment]
And passes Windows 10 as well, no surprise, but.  :)

**Pull request merged**